textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcher.java

```diff
@@ -136,6 +142,13 @@ public ITuple getNextTuple() throws DataFlowException {
         }        
     }
     
+    public void setLimit(int l){
```

Comment:
Do we ever need this `setLimit()` function?  If not, let's remove it.  If we need to keep it, let's use "limit" instead of "l".